sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -57,8 +57,8 @@ trait ExpressionEvalHelper extends GeneratorDrivenPropertyChecks {
     (result, expected) match {
       case (result: Array[Byte], expected: Array[Byte]) =>
         java.util.Arrays.equals(result, expected)
-      case (result: Double, expected: Spread[Double]) =>
-        expected.isWithin(result)
+      case (result: Double, expected: Spread[_]) => // Can't use Spread[Double] b/c of erasure
```

Comment:
Good point, that seems better. Although it can't actually check the generic type, leaving it (with the annotation to flag it and silence the warning) at least better demonstrates the intent. It wouldn't silently pass anyway because of the following cast.